runtime/tchannel_client.go

```diff
@@ -219,8 +219,14 @@ func (c *tchannelOutboundCall) finish(err error) {
 }
 
 func (c *tchannelOutboundCall) logFields() []zapcore.Field {
+	var hostPort string
```

Comment:
@ChuntaoLu can't find an obvious way to write a test for this change. any suggestions?

Reply:
A contrived test case would be to config the client with a hostport that no backend server listens at, but for the sake of this function it's okay to not go that far.

Reply:
Added a unit test instead for the case when call is nil although i couldn't find a way to trivially write unit test for the positive case.